docs/ota-client-guide/modules/ROOT/pages/enable-device-cred-provisioning.adoc

```diff
@@ -101,27 +60,25 @@ scp -P 2222 autoprov.url root@localhost:/var/sota/import/gateway.url
 ====
 You might remember that `credentials.zip` contains a provisioning key shared-credential provisioning. In this case we just need the `autoprov.url` file inside `credentials.zip`. This file contains the URL of your device gateway which is specific to your account.
 ====
-. Copy the device credentials and device gateway root CA certificate to the device's HSM. 
+. Copy the device credentials and device gateway root CA certificate to the device. 
 +
 [source,sh,subs="attributes"]
 ----
-export device_dir=path/to/device/dir
+export device_dir=path/to/device-creds/dir
 scp -P 2222 -pr ${device_dir} root@localhost:/var/sota/import
 ----
 +
 [NOTE]
 ====
-Replace `path/to/device/dir` with the device directory that you noted when  xref:generatetest-devicecert.adoc[generating the device certificate].
-
-For the QEMU simulated HSM, replace `path/to/device/dir` with the credentials directory of the relevant device.
+Replace `path/to/device-creds/dir` with the local directory where you xref:generate-devicecert.adoc[generated the device certificate].
 ====
 +
-. _(Optional)_ When the copy operation has completed, ssh into your device and check the aktualizr log output with the following `systemd` command:
+. _(Optional)_ When the copy operation has completed, SSH into your device and check the aktualizr log output with the following `systemd` command:
```

Comment:
I don't think ssh should be capitalized, generally. `ssh` stands for "Secure Shell". It's usually left lowercase because the name of the tool you use is `ssh`, lowercase, and here we're just using the name of the tool as a verb. But if we want to go for technically correct instead of common usage, it should be SSh, which is ugly and weird.